tests/e2e-pw/specs/merchant/merchant-admin-disputes.spec.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { useMerchant } from '../../utils/helpers';
+
+test.describe( 'Merchant disputes', () => {
+	// Use the merchant user for this test suite.
+	useMerchant();
+
+	test( 'Load the disputes list page', async ( { page } ) => {
+		await page.goto(
+			'/wp-admin/admin.php?page=wc-admin&path=/payments/disputes'
+		);
+
+		// Wait for the disputes table to load.
+		await page
+			.locator( '.woocommerce-table__table.is-loading' )
+			.waitFor( { state: 'hidden' } );
```

Comment:
Nitpick: The `.waitFor()` works great, but the `{ state: 'hidden' }` makes it less obvious. Another alternative would be to change it to `expect().toHaveCount()`.

```suggestion
		await expect(
			page.locator( '.woocommerce-table__table.is-loading' )
		).toHaveCount( 0 );
```

Reply:
Agreed, this is more clear to define what is expected of the test. Updated in https://github.com/Automattic/woocommerce-payments/pull/10123/commits/3d48cdf8dfea693eaf2b271b7a436eef0b7c0da1